front/lib/swr/assistants.ts

```diff
@@ -814,6 +815,37 @@ export function useAgentUsageMetrics({
   };
 }
 
+export function useAgentContextOrigin(params: {
+  workspaceId: string;
+  agentConfigurationId: string;
+  days?: number;
+  version?: string;
+  disabled?: boolean;
+}) {
+  const {
+    workspaceId,
+    agentConfigurationId,
+    days = DEFAULT_PERIOD_DAYS,
+    version,
+    disabled,
+  } = params;
+  const fetcherFn: Fetcher<GetContextOriginResponse> = fetcher;
+  const versionParam = version ? `&version=${encodeURIComponent(version)}` : "";
+  const key = `/api/w/${workspaceId}/assistant/agent_configurations/${agentConfigurationId}/observability/source?days=${days}${versionParam}`;
```

Comment:
nit, nrae i think, but kind of inconsistent to have one param fully stored in a variable and one semi-inlined